addons/web/static/src/core/checkbox/checkbox.scss

```diff
@@ -1,20 +0,0 @@
-// = Checkbox
-// ============================================================================
-
-.custom-control.form-check .form-check-input {
```

Comment:
@stefanorigano Are you ok to remove this file?

Reply:
Yes, we are still looking for them and fixing it case by case. I just want to be sure that this style here isn't necessary anymore for checkboxes.

Reply:
Sure, we can remove 👍